src/languages/en.ts

```diff
@@ -1757,4 +1765,4 @@ export default {
             selectSuggestedAddress: 'Please select a suggested address',
         },
     },
-} satisfies Translation;
```

Comment:
it should satisfies `TranslationBase`, shouldn't?

Reply:
After testing, I agree. I'll add it in a separate PR  👍